app/src/main/java/uc/edu/ozturkkl/traveltracker/ui/main/MainFragment.kt

```diff
@@ -26,7 +28,14 @@ class MainFragment : Fragment() {
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProviders.of(this).get(MainViewModel::class.java)
-        // TODO: Use the ViewModel
+        floatingActionButton3.setOnClickListener{
+            addLocation()
+        }
     }
 
+    private fun addLocation() {
+            Toast.makeText(activity, "Replace this action with the Location Addition Event!", Toast.LENGTH_LONG).show()
```

Comment:
Screen output placeholder that sends a Toast message to the homescreen, showing the button was clicked.